### DISCUSSION TITLE
Questions regarding GitHub Pro

### DISCUSSION THREAD

**Discussion by sanjay-kv** [community/community#135595] [GitHub Education] [answered]
### Hi everyone,

It has mentioned on the main github education page, that when you sign up as a teacher you can avail of GitHub pro for half price, just wondering how can avail of this offer.

**Comment by Deadcoder001** [comment, marked as answer]
You can try a teacher’s account or maybe you can use a student’s id for avail this you just need his/her school/collage id with the year mentioned.
